lib/Monads/trace/Trace_No_Throw.thy

```diff
@@ -0,0 +1,30 @@
+(*
+ * Copyright 2023, Proofcraft Pty Ltd
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ *)
+
+(* Lemmas about no_throw. Usually should have a conclusion "no_throw P m".
+   Includes some monad equations that have no_throw as a main assumption.  *)
+
+theory Trace_No_Throw
+imports
+  \<comment> \<open>Trace_WhileLoopRules
+  Trace_MonadEq_Lemmas\<close>
+  Trace_VCG \<comment> \<open>FIXME: this import replaces the above\<close>
```

Comment:
forgotten FIXME?

Reply:
I originally put that FIXME in as a reminder to keep the same import structure if those other theories were added for the trace monad, but at this stage I'd say that that's unlikely, so I'll remove it.